script/Ciel/Day 9/09_01_CIEL09_6.txt

```diff
@@ -611,12 +612,12 @@ I'm the only one who has the right to address Shiki."
 -- Page 582, Offset 23625. Mods: @n.
 -- 自分が何であるかを理解しない愚か者に、
 // Front loaded because of how the VA timing triggers work
-My only aim here is to impart the truth to an idiot who doesn't seem to understand what she truly is."
+My only aim here is to impart the truth to an idiot who doesn't seem to understand what she truly is.
 }
 [sha:0f206e476d58be802444569a384ffd302cbc1a98]{
 -- Page 582, Offset 23626. Glued.
 -- 現実を教えてあげているだけです」
-%{nothing}
+"
```

Comment:
does this fix the newline in game? I wonder what would cause the glue to mess up the linebreak (or was it just my allscr)

Reply:
Yeah, it does. Not entirely sure what was confusing it, but since this fixed it I didn't bother to investigate.